### PR TITLE
I have added C

### PR DESCRIPTION
**Pull request by Himabindujunnuthula**
I have added C as required for the task.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,6 @@
     <h1>Hello world</h1>
     A
     B
-    
+    C
```
Comment (Bashaboina) [inline index.html]: C should be small case, you have added Big C.

Comment (Bashaboina) [reply index.html]: Ok i am going to change it.

Comment (Bashaboina) [reply index.html]: I have made changes @Bashaboina please check it.

Comment (Himabindujunnuthula) [reply index.html]: I have made changes @Bashaboina please check it.

Comment (Bashaboina) [reply index.html]: Looks good for me @Himabindujunnuthula

**Review by Bashaboina — APPROVED** [review]

**Pull request merged**